akka-remote/src/main/resources/reference.conf

```diff
@@ -179,10 +179,19 @@ akka {
     ### Settings shared by classic remoting and Artery (the new implementation of remoting)
 
     # Using remoting directly is typically not desirable, so a warning will
-    # be shown to make this clear. Set this setting to 'off' to suppress that
+    # be shown to make this clear. Set this setting to 'off' to suppress that.
     # warning.
     warn-about-direct-use = "on"
 
+    # If Cluster is not used, remote watch and deployment are disabled.
+    # To optionally use them while not using Cluster, set to 'on'.
+    use-unsafe-remote-features-without-cluster = "off"
```

Comment:
boolean config properties should typically not be with surrounding `"`, probably not wrong but better without
I see that it's wrong above also `warn-about-direct-use`

Reply:
Fixed the pre-existing `warn-about-direct-use = "on"` and the two new booleans :) thanks.